src/main/java/fi/tkgwf/ruuvi/utils/Utils.java

```diff
@@ -36,24 +34,36 @@ public static String hexToAscii(String hex) {
     }
 
     /**
-     * Gets a MAC address from a space-separated hex string, starting after the
-     * prefix length
-     *
+     * 
+     * Use to read line and confirm if Mac address should be present

```

Comment:
A minor detail to add, now that this method is hcidump-specific, perhaps it should say something like "Use to read a line from hcidump and ..." or something, considering there will be other "input methods" in the future